force_gromacs/chemicals/i_fragment.py

```diff
@@ -0,0 +1,20 @@
+from traits.api import Interface, List, Str, Float
+
+
+class IFragment(Interface):
+    """Contains all input values for each molecular fragment. A
+    fragment is defined as a part of a fragment that may become
+    dissociated (i.e - an ion) and therefore requires its own set
+    of chemical / structural information"""
+
+    #: List of atoms in molecular fragment
+    atoms = List(Str)
+
+    #: Molecular mass of fragment in g / mol
+    mass = Float
+
+    #: Molecular charge of fragment
+    charge = Float
+
+    def get_data_values(self):
+        """Return a list containing all DataValues stored in class"""
```

Comment:
(EOF new line)

Reply:
I think there must be one (otherwise the flake8 check would fail), but GitHub is not showing it